src/js/apis/c2c/ModeratorService.js

```diff
@@ -61,4 +61,20 @@ ModeratorService.prototype.unblockAccount = function (userId) {
   return this.api.post('/users/unblock', { user_id: userId });
 };
 
+/**
+ * Mask/unmask version service
+ */
+ModeratorService.prototype.maskVersion = function ({ documentId, lang, versionId }) {
+  console.log(documentId);
```

Comment:
remove console.log

Reply:
Don't worry, the branch is only at its very early stage :P